frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -247,6 +348,7 @@ export const PersistentVolumeClaimsPage = (props) => {
       canCreate={true}
       rowFilters={filters}
       createProps={createProps}
+      customData={pvcMetrics}
```

Comment:
We are setting state in redux, so there is no need to pass this down as a prop. We just need to retrieve the values from redux on each table row.

Reply:
We could do that but we would have to connect each row to the state. I find this a cleaner approach than to pass a callback as a row. WDYT?

Reply:
I think it's a better pattern to connect to the redux state. You can just use the redux `useSelector` hook directly in each row, or write a custom hook. Redux is specifically designed to avoid "forwarding" props, which is exactly what customData is. It gets forwarded through several components down to each row.

Reply:
Since we wrap it in a callback when a metric update happens the whole row would get updated, but if we pass it as a simple prop only parts of that row will get rerendered(parts that are coming from Prometheus). I think this would be more efficient than connecting the whole row to the store. WDYT?

Reply:
Since the `pvcMetrics` object is being created on every render of `PersistentVolumeClaimsPage` and passed down through the `customData` prop, it will always cause a re-render of the entire table anyway. At least if we connect the rows to redux, we only re-render specific rows that have changed.